wolfssl/internal.h

```diff
@@ -3413,6 +3417,10 @@ typedef struct Arrays {
     byte            secret[SECRET_LEN];
 #endif
     byte            masterSecret[SECRET_LEN];
+#if defined(WOLFSSL_RENESAS_TSIP_TLS) && \
+   !defined(NO_WOLFSSL_RENESAS_TSIP_TLS_SESSION)
+    byte            tsip_masterSecret[80];
```

Comment:
Is there a reason this is larger than the standard master secret length (SECRET_LEN)?

Reply:
Yes. TSIP API, R_TSIP_TlsGenerateMasterSecret(), generates 20 words of master secrete data which is converted by TSIP-specific way.
Just made changed to use a definition instead of the fixed number.